earth2mip/networks/graphcast.py

```diff
@@ -13,35 +13,42 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
I rewrote this from scratch so the diff is ugly, suggest just reviewing the new file.